tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -46,10 +46,11 @@
 from time import sleep
 
 
+from system import execute_wdb_query
```

Comment:
```suggestion
from wazuh_testing.tools.system import HostManager, clean_environment, execute_wdb_query
```

Reply:
I have tried the change but it doesn't work. The `execute_wdb_query` function is at [tests/system/__init__.py](https://github.com/wazuh/wazuh-qa/blob/0fa86091d2bbf35fdf53e3355d5234f5bd714f7e/tests/system/__init__.py#L156) not at [deps/wazuh_testing/tools/system.py](https://github.com/wazuh/wazuh-qa/blob/0fa86091d2bbf35fdf53e3355d5234f5bd714f7e/deps/wazuh_testing/wazuh_testing/tools/system.py#L26)